From 9aa49805ee133cffd54a5709f81d1c2755b1f120 Mon Sep 17 00:00:00 2001
From: Robert Mustacchi <rm@joyent.com>
Date: Tue, 7 Mar 2017 18:51:05 +0000
Subject: [PATCH] OS-5995 Remove old CTF tools Reviewed by: Jerry Jelinek
 <jerry.jelinek@joyent.com> Approved by: Joshua M. Clulow <jmc@joyent.com>

---
 Makefile              | 2 +-
 pkgsrc/onbld/packlist | 1 -
 src/Makefile.defs     | 2 +-
 3 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 67f6f2bd..d5fbfacb 100644
--- a/Makefile
+++ b/Makefile
@@ -64,7 +64,7 @@ CTFBINDIR = \
 	$(ROOT)/projects/illumos/usr/src/tools/proto/*/opt/onbld/bin/i386
 CTFMERGE =	$(CTFBINDIR)/ctfmerge
 CTFCONVERT =	$(CTFBINDIR)/ctfconvert
-ALTCTFCONVERT =	$(CTFBINDIR)/ctfconvert-altexec
+ALTCTFCONVERT =	$(CTFBINDIR)/ctfconvert
 
 SUBDIR_DEFS = \
 	CTFMERGE=$(CTFMERGE) \
diff --git a/pkgsrc/onbld/packlist b/pkgsrc/onbld/packlist
index 7697095d..59047769 100644
--- a/pkgsrc/onbld/packlist
+++ b/pkgsrc/onbld/packlist
@@ -26,7 +26,6 @@
 ./bin/i386/ctfdiff
 ./bin/i386/ctfdump
 ./bin/i386/ctfmerge
-./bin/i386/ctfmerge-altexec
 ./bin/i386/ctfstabs
 ./bin/i386/ctfstrip
 ./bin/i386/cw
diff --git a/src/Makefile.defs b/src/Makefile.defs
index 8ed43227..4b3559e1 100644
--- a/src/Makefile.defs
+++ b/src/Makefile.defs
@@ -42,7 +42,7 @@ GXX =		$(NATIVEDIR)/usr/bin/g++
 CTFBINDIR =	$(ILLUMOS_SRC)/tools/proto/root_i386-nd/opt/onbld/bin/i386
 CTFMERGE =	$(CTFBINDIR)/ctfmerge
 CTFCONVERT =	$(CTFBINDIR)/ctfconvert
-ALTCTFCONV =	$(CTFBINDIR)/ctfconvert-altexec
+ALTCTFCONV =	$(CTFBINDIR)/ctfconvert
 
 SYSINCDIRS =	/usr/include
 SYSLIBDIRS =	/usr/lib /lib
-- 
2.21.0

